site/en/workflows.md

```diff
@@ -1217,19 +1244,146 @@ copies them the a new Output fileGrp, re-generating the PAGE XML from the curren
   </tbody>
 </table>
 
+### Step X: Multi-step
+
+<!-- BEGIN-EVAL sed -n '0,/^## Notes/ p' ./repo/ocrd-website.wiki/Workflow-Guide-multistep.md|sed '$d' -->
+The procecssors in this group do more than one step in an OCR-D workflow. This
+allows these processors to optimize the data flow, e.g. reusing more precise
+internal data structures, instead of relying on the serialization of results by
+a previous step. The downside of these processors is that they are less
+interoperable with the granular processors, removing a certain amount of
+control from the user and making the implementations more complex.
+
+**Note:** All the `ocrd-tesserocr-segment*` processors internally delegate to
+`ocrd-tesserocr-recognize`, so you can replace calls to these task-specific
+processors with calls to `ocrd-tesserocr-recognize` with specific parameters:
+
+<table>
+  <thead><tr><th>processor call</th><th><code>ocrd-tesserocr-recognize</code> parameters</th></tr></thead>
+  <tbody>
+    <tr>
+      <td>ocrd-tesserocr-segment-region -P overwrite_regions true -P crop_polygons</td>
+      <td>ocrd-tesserocr-recognize -P textequiv_level none -P segmentation_level region -P overwrite_segments true -P block_polygons</td>
```

Comment:
`crop_polygons` is [unusable](https://github.com/OCR-D/ocrd_tesserocr/issues/98) at the moment, plus you would need to at least set a _value_ here. How about another param as example:

```suggestion
      <td>ocrd-tesserocr-segment-region -P overwrite_regions true -P find_tables false</td>
      <td>ocrd-tesserocr-recognize -P textequiv_level none -P segmentation_level region -P overwrite_segments true -P find_tables false</td>
```